### PR TITLE
Fix compiler warnings

### PR DESCRIPTION
**Pull request by iilyak**
- Replace use of `erlang:now/0` with `erlang:monotonic_time/0`
- Replace use of `random` module with `rand` module

### CODE REVIEW

File: src/ken_server.erl [hunk 1]
```diff
@@ -509,16 +515,35 @@ resubmit(Delay, DbName) ->
...
     Config = config:get("query_servers") ++ config:get("native_query_servers"),
     [list_to_binary(Lang) || {Lang, _Cmd} <- Config].
 
 config(Key, Default) ->
     config:get("ken", Key, Default).
+
+-ifdef(TEST).
+-include_lib("eunit/include/eunit.hrl").
+
+prune_old_entried_test() ->
```
Comment (jaydoane) [inline src/ken_server.erl]: I think this should be `prune_old_entries_test`?

**Comment by jaydoane** [comment]
Can we also remove this compiler warning?
```
ken/test/ken_server_test.erl:15: Warning: export_all flag enabled - all functions will be exported
```

**Review by jaydoane — CHANGES_REQUESTED** [review]
Prior to this PR, all ken tests pass for me, but with this PR I'm seeing the following issues:
```
module 'ken_server_test'
  undefined
  *** context setup failed ***
**in function ken_server_test:setup_default/0 (test/ken_server_test.erl, line 65)
**error:{badmatch,{error,{badarg,[{ets,new,
                               [ken_workers,[named_table,public,{keypos,2}]],
                               []},
                          {ken_server,init,1,
                                      [{file,"src/ken_server.erl"},
                                       {line,118}]},
                          {gen_server,init_it,2,
                                      [{file,"gen_server.erl"},{line,374}]},
                          {gen_server,init_it,6,
                                      [{file,"gen_server.erl"},{line,342}]},
                          {proc_lib,init_p_do_apply,3,
                                    [{file,"proc_lib.erl"},{line,249}]}]}}}


undefined
*** context setup failed ***
**in function ken_server_test:setup_default/0 (test/ken_server_test.erl, line 63)
**error:{badmatch,{error,{already_started,<0.172.0>}}}


undefined
*** context setup failed ***
**in function ken_server_test:setup_default/0 (test/ken_server_test.erl, line 63)
**error:{badmatch,{error,{already_started,<0.172.0>}}}


undefined
*** context setup failed ***
**in function ken_server_test:setup_default/0 (test/ken_server_test.erl, line 63)
**error:{badmatch,{error,{already_started,<0.172.0>}}}


undefined
*** context setup failed ***
**in function ken_server_test:setup_default/0 (test/ken_server_test.erl, line 63)
**error:{badmatch,{error,{already_started,<0.172.0>}}}


undefined
*** context setup failed ***
**in function ken_server_test:setup_default/0 (test/ken_server_test.erl, line 63)
**error:{badmatch,{error,{already_started,<0.172.0>}}}


undefined
*** context setup failed ***
**in function ken_server_test:setup_default/0 (test/ken_server_test.erl, line 63)
**error:{badmatch,{error,{already_started,<0.172.0>}}}


undefined
*** context setup failed ***
**in function ken_server_test:setup_default/0 (test/ken_server_test.erl, line 63)
**error:{badmatch,{error,{already_started,<0.172.0>}}}


undefined
*** context setup failed ***
**in function ken_server_test:setup_default/0 (test/ken_server_test.erl, line 63)
**error:{badmatch,{error,{already_started,<0.172.0>}}}


undefined
*** context setup failed ***
**in function ken_server_test:setup_config/0 (test/ken_server_test.erl, line 71)
**error:{badmatch,{error,{already_started,<0.172.0>}}}
```

File: src/ken_server.erl (hunk 1 above)
Comment (iilyak) [reply src/ken_server.erl]: fixed

**Comment by iilyak** [comment]
@jaydoane I updated the PR to fix identified issues.

**Review by jaydoane — APPROVED** [review]

**Comment by jaydoane** [comment]
+1 and please remember to squash
```
$ rm -rf src/ken/.eunit && make eunit apps=ken
==> ken (compile)
Compiled src/ken_server.erl
==> rel (compile)
==> couchdb (compile)
==> couchdb (setup_eunit)
==> ken (eunit)
Compiled src/ken_app.erl
Compiled src/ken_sup.erl
Compiled src/ken.erl
Compiled src/ken_event_handler.erl
Compiled test/ken_server_test.erl
Compiled src/ken_server.erl
======================== EUnit ========================
module 'ken'
module 'ken_event_handler'
ken_server: prune_old_entries_test (module 'ken_server')...ok
module 'ken_sup'
module 'ken_app'
module 'ken_server_test'
  ken_server_test:85: set_builder (set_limit returns default)...ok
  ken_server_test:85: set_builder (set_limit keeps set)...ok
  ken_server_test:85: set_builder (set_batch_size returns default)...ok
  ken_server_test:85: set_builder (set_batch_size keeps set)...ok
  ken_server_test:85: set_builder (set_delay returns default)...ok
  ken_server_test:85: set_builder (set_delay keeps set)...ok
  ken_server_test:85: set_builder (set_prune_interval returns default)...ok
  ken_server_test:85: set_builder (set_prune_interval keeps set)...ok
  ken_server_test:81: exception_builder (set_limit exception on zero)...ok
  ken_server_test:81: exception_builder (set_limit exception on negative)...ok
  ken_server_test:81: exception_builder (set_batch_size exception on zero)...ok
  ken_server_test:81: exception_builder (set_batch_size exception on negative)...ok
  ken_server_test:85: set_builder (set_delay no exception on zero)...ok
  ken_server_test:81: exception_builder (set_delay exception on negative)...ok
  ken_server_test:81: exception_builder (set_prune_interval exception on zero)...ok
  ken_server_test:81: exception_builder (set_prune_interval exception on negative)...ok
  ken_server_test:85: set_builder (set_limit reads config)...ok
  ken_server_test:85: set_builder (set_limit keeps set)...ok
  [done in 0.067 s]
=======================================================
  All 19 tests passed.
```

**Comment by rnewson** [comment]
remind me which erlang version the new time functions appeared in and whether couchdb supports older versions?

**Comment by jaydoane** [comment]
@rnewson [OTP 18](http://erlang.org/doc/apps/erts/time_correction.html) and couchdb now supports only OTP 19+

**Pull request merged**